docs/contributing.md

```diff
@@ -119,7 +119,8 @@ Bazel is organized in several parts:
 ## Patch Acceptance Process
 
 1. Discuss your plan and design, and get agreement on our [mailing
-   list](https://groups.google.com/forum/#!forum/bazel-discuss).
+   list](https://groups.google.com/forum/#!forum/bazel-discuss) or in 
+   a github issue.
```

Comment:
This is maybe contentious, but I suspect that you will get people filing proposals for ideas in github issues, and it is not an unreasonable forum for such discussion.